pipelines/build/regeneration/evaluation_pipeline_generator.groovy

```diff
@@ -48,7 +48,7 @@ node('worker') {
         }
 
         timestamps {
-            def validVersion = [8, 11, 17, 20]
+            def retiredVersions = [9, 10, 12, 13, 14, 15, 16, 18, 19]
```

Comment:
Hmm, do we need a shared properties ofr config file to track this sort of thing across our logic?

Reply:
Perhaps yes. Not quite sure what a "shared properties ofr config file" is ? @karianna 
Using the jenkins-helper might be possible, possibly add to : https://github.com/adoptium/jenkins-helper/blob/master/vars/Constants.groovy

Reply:
think we also need to update documentation (related to post release work) when we retire a release.